web/thesauruses/_meta/trees.json

```diff
@@ -0,0 +1,94 @@
+{
+  "meta": {
+    "structure": "trees",
+    "structure_name": "Trees"
+  },
+  "categories": {
+    "Creating Basic Tree Objects":{
+        "structure_of_node": "What data structure do you create the tree node with?",
+        "information_inside_node": "Information stored inside of a node",
+        "create_node_class": "Creating the node class or struct",
+        "create_a_node_object": "Creating an object of the node class or struct",
+        "create_tree_class": "Creating the tree class",
```

Comment:
Since this is a language reference tool, we wouldn't create the tree/node classes, we would be using whatever pre-existing tree class(es) that exists in whatever language. So this might just be `name_of_tree_class` and `instantiate_tree_object` (as a tree probably has some type of generic object attached to it in Java/C++/Rust/etc.). 

Maybe rework this part so it's not implying you're custom-making your own trees.